sw/airborne/modules/calibration/mag_calib_ukf.h

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (C) w.vlenterie
+ *
+ * This file is part of paparazzi
+ *
+ * paparazzi is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2, or (at your option)
+ * any later version.
+ *
+ * paparazzi is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with paparazzi; see the file COPYING.  If not, see
+ * <http://www.gnu.org/licenses/>.
+ */
+/**
+ * @file "modules/calibration/mag_calib_ukf.h"
+ * @author w.vlenterie
+ * Calibrate magnetometer using UKF
+ */
+
+#ifndef MAG_CALIB_UKF_H
+#define MAG_CALIB_UKF_H
+
+#ifdef AHRS_FLOAT_CMPL_WRAPPER_H
```

Comment:
this is brittle and error prone... and probably wont even work as intended in a lot of cases where file is never included...